meinberlin/assets/scss/components/_list_item.scss

```diff
@@ -15,7 +15,7 @@
         outline: none;
     }
 
-    &:hover {
+    &:hover:not(.no-hover) {
```

Comment:
what does this do?

Reply:
it prevents the tile from having a hover state, which is weird/confusing because if there is a button. we have a doubled hovering effect. E.g. if the mouse is over the tile, it gets darker (the button too).  @mcastro-lqd maybe you have an idea on this?

Reply:
just checked with @mcastro-lqd and from a design's perspective a hover state without action does not really make sense. so preferably to be removed everywhere. i will do it when coming back to it for "making it possible for all buergerhaushalt variants".

Reply:
We should make the whole item clickable when it has a hover effect, will add an issue